radis/lbl/loader.py

```diff
@@ -1821,6 +1831,26 @@ def load_and_concat(files):
                 )
             )
 
+        id_set = df.id.unique()
+
+        if len(id_set) != 1:  # only 1 molecule supported ftm
+            raise NotImplementedError(
+                "Only 1 molecule at a time is currently supported "
+                + "in SpectrumFactory. Use radis.calc_spectrum, which "
+                + "calculates them independently then use MergeSlabs"
+            )
+
+        df.drop("id", axis=1, inplace=True)
+        df_metadata.append("id")
+        df.attrs["id"] = id_set[0]
+
+        isotope_set = df.iso.unique()
+
+        if len(isotope_set) == 1:
+            df.drop("iso", axis=1, inplace=True)
+            df_metadata.append("iso")
+            df.attrs["iso"] = isotope_set[0]
```

Comment:
Are `df_metadata` and `df.attrs` two different storage ? Let's keep one only (maybe future PR , this one is huge already!).  If you agree just open a quick Refactor issue from this comment.

Reply:
Isn't df_metadata just a manually created dictionary in `loader.py`? I appended `iso` to this dictionary as well to make it consistent with the few other parts of the code where df_metadata is used (it is only used at one place while transferring metadata). I think we should drop this dictionary and just stick with `df.attrs` instead. 

I am creating a refactor issue.

Reply:
(for reference : https://github.com/radis/radis/issues/300 )